media-nep/src/main/java/nl/vpro/nep/service/impl/NEPItemizeServiceImpl.java

```diff
@@ -92,12 +96,16 @@ public void shutdown() throws IOException {
         }
     }
 
-    @SneakyThrows
-    protected NEPItemizeResponse itemize(@NonNull NEPItemizeRequest request, String itemizeUrl, Supplier<String> itemizeKey) {
+    protected NEPItemizeResponse itemize(@NonNull NEPItemizeRequest request, String itemizeUrl, Supplier<String> itemizeKey) throws NEPException {
         String playerUrl = itemizeUrl + "/api/itemizer/job";
         log.info("Itemizing {} @ {}", request, playerUrl);
         HttpClientContext clientContext = HttpClientContext.create();
-        String json = Jackson2Mapper.getLenientInstance().writeValueAsString(request);
+        String json = null;
+        try {
+            json = Jackson2Mapper.getLenientInstance().writeValueAsString(request);
+        } catch (JsonProcessingException e) {
+            throw new NEPException(e, "Exception from a NEP function call");
```

Comment:
Zorg dat de oorzaak in de exceptie zit. Tenminste de exception cause zelf. Maar het zou in dit geval bijv. ook handit zijn om het request object er bij te hebben. 

Anders is het vrees ik niet meer te debuggen als het voorkomt.

Reply:
Oh, dat zit er wel.